src/bytecodes/config.json

```diff
@@ -30,13 +30,5 @@
     "features": [
       "ENFORCED_BINARY"
     ]
-  },
```

Comment:
Can we add this back in

Reply:
This file is autogenerated by scripts/updateDeployedBytecodes after each yarn install. Something must change in dxdao contracts for this to be removed. The reason must be that script couldn't find the json inside artifacts.